frontend/src/components/periodical-evaluation/references/PeriodicalEditionReferenceInput.vue

```diff
@@ -174,7 +179,11 @@ function addReference(decision: RelatedDocumentation) {
 
   validateRequiredInput(newReference)
 
-  if (validationStore.isValid()) {
+  // Todo validate also literature input
+  if (
+    validationStore.isValid() ||
+    reference.value.referenceType == "literature"
```

Comment:
@leonie-koch Ah, I found the reason for [this](https://github.com/digitalservicebund/ris-backend-service/pull/2370/files#r1846555995)